test/schedule/data_test.exs

```diff
@@ -1762,6 +1762,250 @@ defmodule Schedule.DataTest do
              } = Data.run_from_hastus(run_key, activities, trips, %{}, %{}, %{})
     end
 
+    test "allows as directed trips and regular trips in the same piece named like a regular block" do
```

Comment:
This test passed without the changes here (because the block name doesn't have `wad`/`rad` in it).